eng/common/pipelines/templates/steps/sparse-checkout.yml

```diff
@@ -24,15 +24,13 @@ steps:
     - pwsh: |
         git clone --no-checkout --filter=tree:0 git://github.com/${{ repo.Name }} .
         git sparse-checkout init
-        git sparse-checkout set eng
+        $paths = ('${{ convertToJson(parameters.Paths) }}' | ConvertFrom-Json) -Join ' '
```

Comment:
Interesting pattern. We might be able to use this to move other things into inline loop in the script block instead of having many devops steps.

Reply:
Yes it's very useful. I could see some of our release configs being made much simpler this way.